tests/e2e/login/privacy-policy.wdio-spec.js

```diff
@@ -0,0 +1,128 @@
+const utils = require('../../utils');
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const privacyPolicyPage = require('../../page-objects/privacy-policy/privacy-policy.wdio.page');
+const userFactory = require('../../factories/cht/users/users');
+const privacyPolicyFactory = require('../../factories/cht/settings/privacy-policy');
+const placeFactory = require('../../factories/cht/contacts/place');
+
+describe('Privacy policy', () => {
+  const englishTexts = privacyPolicyFactory.english;
+  const frenchTexts = privacyPolicyFactory.french;
+  const offlineUser = userFactory.build();
+  const onlineUser = userFactory.build({
+    username: 'online',
+    roles: ['program_officer'],
+    place: {
+      _id: 'hc2',
+      type: 'health_center',
+      name: 'Health Center 2',
+      parent: 'dist1'
+    },
+    contact: {
+      _id: 'fixture:user:onlineuser',
+      name: 'onlineuser'
+    }
+  });
+
+  const parent = placeFactory.place().build({_id:'dist1',type:'district_hospital'});
+
+  before(async () => {
+    await utils.saveDocs([parent, privacyPolicyFactory.privacyPolicy().build()]);
+    await utils.createUsers([ onlineUser, offlineUser]);
+  });
+
+  describe('for an online user', () => {
+    it('should show the correct privacy policy on login', async () => {
+      // After first login, check that privacy policy was prompted to user
+      await loginPage.login(onlineUser.username, onlineUser.password);
+      await browser.waitUntil(async () => {
+        const wrapperText = await (await privacyPolicyPage.privacyWrapper()).getText();
+        return wrapperText.includes(englishTexts.header) && wrapperText.includes(englishTexts.paragraph);
+      }, 'Timed out waiting for English Online Privacy Policy to Display');
+
+      // After accepting, no privacy policy on next load
+      await privacyPolicyPage.acceptPrivacyPolicy();
+      await browser.url('/');
+      expect(await privacyPolicyPage.privacyWrapper()).not.toBeDisplayed();
+
+      // Check display when loading privacy policy page
+      await privacyPolicyPage.goToPrivacyPolicyConfig();
+      await browser.waitUntil(async () => {
+        const wrapperText = await (await privacyPolicyPage.privacyConfig()).getText();
+        return wrapperText.includes(englishTexts.header) && wrapperText.includes(englishTexts.paragraph);
+      }, 'Timed out waiting for english online privacy to display');
+
+      // No privacy policy on 2nd login
+      await browser.reloadSession();
+      await browser.url('/');
+      await loginPage.login(onlineUser.username, onlineUser.password);
+      expect(await privacyPolicyPage.privacyWrapper()).not.toBeDisplayed();
+
+
+      // After login in french, check that privacy policy was prompted to user again
+      await browser.reloadSession();
+      await browser.url('/');
+      await loginPage.login(onlineUser.username, onlineUser.password, 'fr');
```

Comment:
Could we change this so we only change the user's language and refresh? 
Like 
```suggestion
      await browser.setCookie({ locale: 'fr' });
      await browser.refresh();
```

This way we're also testing what happens when the language is changed without logging in again.

Reply:
Yeah, I'll add an additional test case for that.  I think this test needs to be split up into more separate tests into 2 that do online and offline. I'll resubmit.